metricflow/test/model/validations/test_configurable_rules.py

```diff
@@ -0,0 +1,31 @@
+from metricflow.model.model_validator import ModelValidator
+from metricflow.model.objects.materialization import Materialization
+from metricflow.model.validations.materializations import ValidMaterializationRule
+from metricflow.model.objects.user_configured_model import UserConfiguredModel
+from metricflow.test.test_utils import model_with_materialization
+
+
+def test_can_configure_model_validator_rules(simple_model__pre_transforms: UserConfiguredModel) -> None:  # noqa: D
+    model = model_with_materialization(
+        simple_model__pre_transforms,
+        [
+            Materialization(
+                name="foobar",
+                metrics=["invalid_bookings"],
+                dimensions=["ds"],
+            )
+        ],
+    )
+
+    # confirm that with the default configuration, an issue is raised
+    issues = ModelValidator().validate_model(model).issues
+    assert len(issues) == 1, f"ModelValidator with default rules had unexpected number of issues {issues}"
+
+    # confirm that with an empty configuration, no issue is raised
+    issues = ModelValidator(rules=[]).validate_model(model).issues
+    assert len(issues) == 0, f"Empty rules list configured ModelValidator returned issues {issues}"
```

Comment:
This feels like it belongs in a different test case.

Also, do we want to allow empty validation runs? If not, let's add an assertion and then split this into a test case that makes sure the exception gets thrown correctly.